decentraland/unity-explorer
# fix: Emotes menu is overlapping the Wearables menu (#1981)

The E key, which brings you to the emote slots configuration menu, was enabled while the emotes wheel menu was not ready to be closed yet (the task that is awaiting for the controller to close is not listening to cancellations or completion yet). Now the key is enabled after the menu is ready.

## What does this PR change?

Now the E key is not enabled until the emote wheel is ready.

## How to test the changes?

1. Launch the explorer
2. Press B and then E alternatively, as fast as possible.

The emote wheel and the smote slots configuration menu should not overlap, if one is present, the other should not be.
